resources/qml/RoomMembers.qml

```diff
@@ -9,7 +9,7 @@ import QtQuick.Layouts 1.12
 import QtQuick.Window 2.12
 import im.nheko 1.0
 
-ApplicationWindow {
+Dialog {
```

Comment:
Why did you change this?

Reply:
I thought that was what you meant. I must have misunderstood. What exactly did you want as far as making this a dialog?

Reply:
The other windows have type: Qt.Dialog or so. We did that because otherwise the windows didn't get positioned correctly. Although we can probably remove that, once everything is qml and we can set a parent window?